views/openacademy_wizard_views.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
- Missing this file on the manifest. As I mentioned on last meeting, it's mandatory to test everything before requesting review
- All wizard files (including views) should be placed under the "wizards" directory, as you did for the model